tensorflow/lite/micro/kernels/activations.cc

```diff
@@ -77,6 +78,157 @@ inline void Relu6Quantized(Q lower, Q upper, const RuntimeShape& input_shape,
   }
 }
 
+inline std::int32_t RoundingDivideByPOT(std::int32_t numerator, int exponent) {
```

Comment:
Please avoid inlining code from tensorflow/lite/kernels/internal/reference/reference_ops.h.

Instead, take methods you want to share and create a new header, like we have done for other operators.  See conv.h for an example.  Add the new header to reference_ops.h and depend only on reference/hard_swish.h or reference/activations.h in the TFMicro kernel.

Reply:
Moving RoundingDivideByPOT to new reference/activations.h. I thought TFLu should avoid using gemmlowp, that is why I added RoundingDivideByPOT.